### PR TITLE
Fix/tr 199 flat radio navigation

### PR DESCRIPTION
**Pull request by lecosson**
Related to https://oat-sa.atlassian.net/browse/TR-199

- enable flat navigation for radio inputs on NFER tests with REBL theme
- load modules with correct namespace to avoid double bundling

Required: 
- https://github.com/oat-sa/extension-tao-testqti/pull/1980
- https://github.com/oat-sa/extension-tao-nfer/pull/254

How to check:
 - create delivery with more than one test item and REBL theme enabled
 - check the inputs are accesible via tab key

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-test-runner-qti",
-    "version": "2.18.7",
+    "version": "2.18.8",
```
Comment (btamas) [inline package.json:3]: Version bump is not necessary.

**Review by btamas — CHANGES_REQUESTED** [review]

**Review by btamas — CHANGES_REQUESTED** [review]
The `itemNavigation.js` is not loaded on my side, so it takes no effect.

**Review by btamas — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by jsconan — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Comment by jsconan** [comment]
Units test for regarding the keyNavigation are passing successfully. However, there is something preventing the full test suite to complete. But it is not related to this PR.

**Review by meefox — APPROVED** [review]

**Pull request merged**